custom_components/frigate/views.py

```diff
@@ -164,7 +164,11 @@ async def _handle_request(
             request.method,
             url,
             headers=source_header,
-            params=request.query,
+            params={
+                k: v
+                for k, v in request.query.items()
+                if k != "authSig" or isinstance(self, VodProxyView)
```

Comment:
This breaks the abstraction. How about we add a `_getParameters` method or equivalent, with a default that blocks out authSig, and then just override that in `VodProxyView`?

Reply:
Sure, that sounds good. I'll make the change in a bit.